packages/foam-vscode/src/features/preview-navigation.ts

```diff
@@ -60,7 +75,7 @@ export const markdownItWithFoamTags = (
     replace: (tag: string) => {
       try {
         const resource = workspace.find(tag);
-        if (resource == null) {
+        if (resource === null) {
```

Comment:
the reason why we use `==` instead of `===` is because it also catches `undefined`, as opposed to just strictly `null`.

But the right approach here would be to use the utility function `isNone`